### PR TITLE
[MIG] website_event_track: Nothing to do

### PR DESCRIPTION
**Pull request by cubells**
cc @Tecnativa

### CODE REVIEW

File: addons/website_event_track/migrations/11.0.1.0/openupgrade_analysis_work.txt [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+# Nothing to do
+
+website_event_track / event.track              / speaker_ids (many2many)       : DEL relation: res.partner
+# Nothing to do
+
+website_event_track / event.track              / stage_id (many2one)           : NEW relation: event.track.stage, required: required, req_default: function
+# Nothing to do
+
+website_event_track / event.track              / state (selection)             : DEL required: required, selection_keys: ['announced', 'cancel', 'confirmed', 'draft', 'published', 'refused'], req_default: function
+# Nothing to do
```
Comment (MiquelRForgeFlow) [inline addons/website_event_track/migrations/11.0.1.0/openupgrade_analysis_work.txt:13]: There exists a map between the `state` and the `stage_id`. Each event track that had a state should have now the corresponding stage_id. There exists a stage_id for each state.

File: addons/website_event_track/migrations/11.0.1.0/openupgrade_analysis_work.txt [hunk 2]
```diff
@@ -0,0 +1,51 @@
+---Fields in module 'website_event_track'---
+website_event_track / event.track              / activity_ids (one2many)       : NEW relation: mail.activity
+website_event_track / event.track              / kanban_state (selection)      : NEW required: required, selection_keys: ['blocked', 'done', 'normal'], req_default: function
+# Nothing to do
+
+website_event_track / event.track              / speaker_ids (many2many)       : DEL relation: res.partner
+# Nothing to do
```
Comment (MiquelRForgeFlow) [inline addons/website_event_track/migrations/11.0.1.0/openupgrade_analysis_work.txt:7]: if the event track record doesn't have a partner_id, then you should fill it with one of the speaker_ids

**Review by MiquelRForgeFlow — CHANGES_REQUESTED** [review]

**Comment by MiquelRForgeFlow** [comment]
Superseded by https://github.com/OCA/OpenUpgrade/pull/1874.

**Comment by pedrobaeza** [comment]
Superseeded by #1874

**Pull request closed without merging**